product_docs/docs/eprs/7/08_xdb_cli/03_xdb_cli_commands/04_encrypt_password.mdx

```diff
@@ -22,16 +26,188 @@ The file containing the text to be encrypted.
 
 The file containing the encrypted form of the text from `infile`.
 
+`-passwordEncryptionMethod` 
+
+Defines the algorithm used to encrypt the password (`DES-MD5` or `AES-SHA256`). If the flag is omitted, the algorith defaults to `DES-MD5`. To override the default `DES-MD5` encryption with the FIPS-compliant `AES-SHA256` encryption, use `-passwordEncryptionMethod AES-SHA256`.
+
+`AES-SHA256` offers stronger protection and supports FIPS, whereas `DES-MD5` is backwards compatible with previous versions.
+
+!!!note
+Ensure you use the same encryption method in the publication and subscription servers of a replication cluster.
+!!!
+
 ## Examples
 
-The file `infile` contains the word `password`.
+### Encrypt a password with DES-MD5
+
+1.  Create a text file with your password using the editor of your choice. Here, `infile` contains the word `password`.
+
+    ```
+    password
+    ```
+
+1.  Execute the `encrypt` command to produce a file named `pwdfile`.
+
+    ```
+    $ java -jar edb-repcli.jar -encrypt -input ~/infile -output ~/pwdfile
+    ```
+
+    The content of file pwdfile contains the encrypted form of `password`, namely `ygJ9AxoJEX854elcVIJPTw==`.
+
+1.  Copy and paste the encrypted password into the Replication Server configuration file: 
+
+    ```shell
```

Comment:
should *not* be shell language - I guess `ini` would be the closest in spirit to the format of these config files.